ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -0,0 +1,338 @@
+"""
+Module responsible for translating g2p data into GA4GH native
+objects.
+"""
+
+import ga4gh.protocol as protocol
+import ga4gh.exceptions as exceptions
+from urlparse import urlparse, urlunsplit
+import rdflib
+
+
+class G2PDataset:
+    """
+    an rdf object store
+    """
+
+    # keep the graph as a static variable
+    rdfGraph = None
+
+    def __init__(self, sources):
+
```

Comment:
Remove the whitespace above and below docstring. I'm not really any the wiser about what sources is though: what does it mean, and what does it contain?